functions/models/src/functions/dismissMessages.ts

```diff
@@ -0,0 +1,22 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { z } from 'zod'
+import { optionalish, optionalishDefault } from '../helpers/optionalish.js'
+
+export const dismissMessagesInputSchema = z.object({
+  userId: optionalish(z.string()),
+  messageIds: z.array(z.string()).optional(),
```

Comment:
Just for consistency with the other properties, I'd use optionalish here as well.